scripts/development/start-contenta_jsonapi.sh

```diff
@@ -30,15 +23,18 @@ cd ${DOCROOT}
 echo -e "\n"
 DRUSH="$DEST_DIR/bin/drush"
 
+WEB_HOST=${WEB_HOST:-127.0.0.1}
```

Comment:
What about keeping around the ```.env``` | ```.env.local``` files? I kinda like its way to specify environment variables.